api/envoy/admin/v2alpha/listeners.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package envoy.admin.v2alpha;
+
+option java_outer_classname = "ListenersProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.admin.v2alpha";
+
+import "envoy/api/v2/core/address.proto";
+
+// [#protodoc-title: Listeners]
+
+// Admin endpoint uses this wrapper for `/listeners` to display listener status information.
+// See :ref:`/listeners <operations_admin_interface_listeners>` for more information.
+message Listeners {
+  // List of listener statuses.
+  repeated ListenerStatus listener_statuses = 1;
+}
+
+// Details an individual listener's current status.
+message ListenerStatus {
+  // Name of the listener
+  string name = 1;
+
+  // The actual local address that the listener is listening on. If the listener was configured
+  // to listen on port 0 on Linux, then this address has the port that was allocated by the OS.
```

Comment:
er...it is "OS" already?

Reply:
or did you mean s/Linux/OS ?

Reply:
I was suggesting to capitalize "OS".

Reply:
Ah ok, unless I'm missing something, it's already capitalized. I did reword this docstring so that it more closely matches the admin docs for `/listeners`.

Reply:
I think I'm reading challenged :) Yes, looks good.